exporter/clickhouseexporter/README.md

```diff
@@ -169,5 +311,66 @@ CREATE TABLE otel_logs
         SETTINGS index_granularity = 8192, ttl_only_drop_parts = 1;
 ```
 
+### Traces
+
+```clickhouse
+CREATE TABLE otel.otel_traces
+(
+    `Timestamp`     DateTime64(9) CODEC (Delta(8), ZSTD(1)),
+    `TraceId`       String CODEC (ZSTD(1)),
+    `SpanId`        String CODEC (ZSTD(1)),
+    `ParentSpanId`  String CODEC (ZSTD(1)),
+    `TraceState`    String CODEC (ZSTD(1)),
+    `SpanName` LowCardinality(String) CODEC (ZSTD(1)),
+    `SpanKind` LowCardinality(String) CODEC (ZSTD(1)),
+    `ServiceName` LowCardinality(String) CODEC (ZSTD(1)),
+    `ResourceAttributes` Map(LowCardinality(String), String) CODEC (ZSTD(1)),
+    `SpanAttributes` Map(LowCardinality(String), String) CODEC (ZSTD(1)),
+    `Duration`      Int64 CODEC (ZSTD(1)),
+    `StatusCode` LowCardinality(String) CODEC (ZSTD(1)),
+    `StatusMessage` String CODEC (ZSTD(1)),
+    `Events.Timestamp` Array(DateTime64(9)) CODEC (ZSTD(1)),
+    `Events.Name` Array(LowCardinality(String)) CODEC (ZSTD(1)),
+    `Events.Attributes` Array(Map(LowCardinality(String), String)) CODEC (ZSTD(1)),
+    `Links.TraceId` Array(String) CODEC (ZSTD(1)),
+    `Links.SpanId` Array(String) CODEC (ZSTD(1)),
+    `Links.TraceState` Array(String) CODEC (ZSTD(1)),
+    `Links.Attributes` Array(Map(LowCardinality(String), String)) CODEC (ZSTD(1)),
```

Comment:
Does it represent 7 independent arrays? Can you please clarify why this data model is chosen? Why can't we use a nested data structure instead?

Reply:
> Does it represent 7 independent arrays?

No, it just copy from the result of `show create table`. for clearly, I have update it to nested data structure